### PR TITLE
Update booking_lower_lim with single line code

### PR DESCRIPTION
**Pull request by priyanshukdc**
try catch finally not needed for updating booking_lower_lim

### CODE REVIEW

File: contentScript.js [hunk 1]
```diff
@@ -29,16 +29,7 @@ if (center_prefs.length === 0) {
...
-try {
-  booking_lower_lim = parseInt(minavailability);
-} catch (e) {
-  booking_lower_lim = 1;
-} finally {
-  if (isNaN(booking_lower_lim)) {
-    booking_lower_lim = 1;
-  }
-}
+let booking_lower_lim = minavailability ? parseInt(minavailability) : 1;
```
Comment (sushrut111) [inline contentScript.js:32]: What if
```
minavailability = "abc";
```

**Review by sushrut111 — CHANGES_REQUESTED** [review]

File: contentScript.js (hunk 1 above)
Comment (priyanshukdc) [reply contentScript.js:32]: For such case, let booking_lower_lim = isNaN(parseInt(minavailability)) ? 1 :parseInt(minavailability);
By the way, there will no such case where minavailability will have String other than a Number. In my opinion, checking NaN for such cases is redundant & should be avoided.

Comment (sushrut111) [reply contentScript.js:32]: If someone, out of experimentation edits the local storage directly, the script would stop working.

Comment (sushrut111) [reply contentScript.js:32]: > For such case, let booking_lower_lim = isNaN(parseInt(minavailability)) ? 1 :parseInt(minavailability);
> By the way, there will no such case where minavailability will have String other than a Number. In my opinion, checking NaN for such cases is redundant & should be avoided.

Readability?

Comment (priyanshukdc) [reply contentScript.js]: I have changed a bit of code and made it readable. Also there was unnecessary catch which was removed as parseInt will never throw error. And thus no need of try and finally.

**Comment by sushrut111** [comment]
Looks good. Please merge main branch into this.

**Comment by priyanshukdc** [comment]
Merged. Accept changes.

**Review by sushrut111 — APPROVED** [review]

**Comment by sushrut111** [comment]
Thank you for your contribution.

**Pull request merged**